tiflash/tiflash-configuration.md

```diff
@@ -156,7 +156,8 @@ delta_index_cache_size = 0
 [profiles]
 
 [profiles.default]
-    ## 存储引擎的 segment 分裂是否使用逻辑分裂。使用逻辑分裂可以减小写放大，但是会造成一定程度的硬盘空间回收不及时。默认为 false。不建议修改默认选项。
+    ## [deprecated] 存储引擎的 segment 分裂是否使用逻辑分裂。使用逻辑分裂可以减小写放大，但是会造成一定程度的硬盘空间回收不及时。默认为 false。
+    ## 注意 6.2 版本 logical split 与其他 feature 存在兼容性问题，不建议将该配置项设置为 true。
```

Comment:
```suggestion
    ## 注意 logical split 在 v6.2 版本有已知问题，强烈不建议将该配置项设置为 true。
```

Reply:
Not mark "deprecated" since we don't remove the config item from code. Just mark as not recommended.